lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -115,8 +115,8 @@ <h2>Byte efficiency tester page</h2>
   <!-- PASS(optimized): image is vector -->
   <!-- PASS(webp): image is vector -->
   <!-- PASS(responsive): image is vector -->
-  <!-- FAIL(offscreen): image is offscreen -->
```

Comment:
this seems to be the only failing case checking the `offscreen` audit in our smoke tests. Can we keep this?

Reply:
+1 to keeping this and adding a new, specific lazy loading images test image